src/peergos/server/tests/simulation/Simulator.java

```diff
@@ -696,7 +711,7 @@ public SimulationSupplier(Map<Simulation, Double> probabilities, Random random)
         int seed = simulatorArgs.getInt("random-seed", 1);
         int nUsers = simulatorArgs.getInt("n-users", 3);
         int meanFileLength  = simulatorArgs.getInt("mean-file-length", 256);
-        boolean randomizeFriendNetwork = simulatorArgs.getBoolean("randomize-friend-network", true);
+        boolean randomizeFriendNetwork = simulatorArgs.getBoolean("randomize-friend-network", false);
```

Comment:
what happens with this true? does it still verify?

Reply:
Currently if the friend network is randomised the code does not take the relationshsips into account when granting sharing access.